### DISCUSSION TITLE
Trying to build a general relaxation function through MACE, need help with output!

### DISCUSSION THREAD

**Discussion by ceir13** [ACEsuit/mace#551] [Q&A] [answered]
Hi, all. I am currently trying to build a general function to relax multiple files with the MACE calculator, however, when I pass the files through either they converge (which is what I need), they run indefinitely, or the Fmax zeroes out after one step. Any help would be appreciated on this problem.

<img width="1126" alt="Screenshot 2024-08-13 at 12 32 18 PM" src="https://github.com/user-attachments/assets/1935ba60-d893-4261-b8ae-d7c9bae27ddd">

<img width="1119" alt="Screenshot 2024-08-13 at 12 38 15 PM" src="https://github.com/user-attachments/assets/00194ce1-2632-4ade-94fd-55a0762d9ad6">

**Comment by gabor1** [comment]
I always recommend to use the PreconLBFGS, it's a more robust optimiser, even when your system is small and it doesn't do preconditioning

**Comment by ceir13** [comment, marked as answer]
Edit: the answer was deleting the restart argument in the dyn function.
